dev.yml

```diff
@@ -0,0 +1,28 @@
+name: argo-admin
+
+type: node
+
+up:
+  - git_hooks:
+      pre-commit: ./scripts/prettier.sh
+  - railgun
+  - node:
+      version: v12.16.3
+      yarn: true
+      packages:
+        - .
+
+commands:
+  build: yarn run build
+  console: node
+  new-extension-point: node ./scripts/argo-admin/new-extension-point
+  nuke: yarn run nuke && rm -rf ./node_modules
+  server: yarn run server:admin --host=https://argo-playground.myshopify.io
+  simulator: yarn run simulator:admin
+  test: yarn run test
+  build-consumer: yarnpkg build && ./scripts/build-consumer.sh
+  restore-consumer: ./scripts/restore-consumer.sh
+  clean: git clean -xdf ./packages
+
+open:
+  app: https://argo-playground.myshopify.io
```

Comment:
I was wondering about having `dev.yml` in public repos 🤔  ... but it seems fine (e.g. [`polaris-react`](https://github.com/Shopify/polaris-react/blob/main/dev.yml) and [`quilt`](https://github.com/Shopify/quilt/blob/main/dev.yml) are okay with it).